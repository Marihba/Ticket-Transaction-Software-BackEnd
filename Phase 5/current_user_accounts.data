admin           AA 000000.00
standard        FS 000400.00
buyer           BS 070000.10
seller          SS 000000.00
this User is OG FS 123456.89
Nami            BS 800000.00
END                         
