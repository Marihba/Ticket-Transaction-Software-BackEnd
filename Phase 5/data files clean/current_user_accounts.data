admin           AA 000000.00
standard        FS 000200.00
buyer           BS 070000.10
seller          SS 000370.35
END                         
