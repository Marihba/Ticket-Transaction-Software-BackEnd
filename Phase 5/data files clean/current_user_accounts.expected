admin           AA 000000.00
standard        FS 000099.98
buyer           BS 070000.10
seller          SS 000470.37
this User is OG FS 123456.89
Nami            BS 800000.00
END                         
